gtsfm/scene_optimizer.py

```diff
@@ -76,6 +78,7 @@ def __init__(
         self.feature_extractor = feature_extractor
         self.two_view_estimator = two_view_estimator
         self.multiview_optimizer = multiview_optimizer
+        self.dense_multiview_optimizer = dense_multiview_optimizer
```

Comment:
The roles of the `MVSBase` and the `MultiviewOptimizer` are very different; naming it `dense_multiview_optimizer` makes it seem like it's some instantiation of a `MultiviewOptimizer`. I think we should change the name to something more representative

Reply:
Agreed that the name is perhaps not ideal. The current name comes from some naming we discussed a few months ago in our meetings, and is in the current graph figure:

![Screen Shot 2021-11-10 at 11 46 35 AM](https://user-images.githubusercontent.com/16724970/141155953-08321061-f0b4-460d-8be1-f6e3125b5769.png)
 
Definitely open to better names also, though, if you have suggestions.